Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
+                } else if (typeof value === 'string') {
+                    this._show = new Expression(value);
+                } else if (defined(value.conditions)) {
+                    this._show = new ConditionsExpression(value);
+                } else if (defined(value.expression) || defined(value.conditionsExpression)) {
+                    this._show = value;
+                } else {
+                    this._show = undefined;
+                    return;
+                }
                 this._showShaderFunctionReady = false;
-                this._show = value;
             }
         },
```

Comment:
I having some trouble with a test, if I use setter in `setup`. In case I use new setter to initialize all variables following test fail: `return undefined shader functions when the style is empty`. But I think existing tests are wrong in this case. If e.g. no color is defined `DEFAULT_JSON_COLOR_EXPRESSION` will be used. So I don't understand why the test is expecting an undefined result instead of a shader generated from value `DEFAULT_JSON_COLOR_EXPRESSION`.